src/GuiBase/Utils/KeyMappingManager.hpp

```diff
@@ -45,9 +45,14 @@ class RA_GUIBASE_API KeyMappingManager
     static std::string enumNamesFromKeyboardModifiers( const Qt::KeyboardModifiers& modifiers );
 
     Context getContext( const std::string& contextName ) {
-        return m_contextNameToIndex[contextName];
+        // use find so that it do not insert invalid context
+        auto itr = m_contextNameToIndex.find( contextName );
+        if ( itr != m_contextNameToIndex.end() ) return m_contextNameToIndex[contextName];
+        return Context{};
     }
     KeyMappingAction getActionIndex( const Context& context, const std::string& actionName ) {
+        if ( context >= m_actionNameToIndex.size() || context.isInvalid() )
+            return KeyMappingAction{};
```

Comment:
Same as above ? A specific `ErrorAction` ?

Reply:
Same as above, default ctor is invalid.